referencia/webpay/README.md

```diff
@@ -1788,3 +1788,100 @@ username  <br> <i> xs:string </i> | Nombre de usuario, del cliente, en el sistem
+
+### Captura Diferida Webpay OneClick Mall
+> Los SDKs no soportan aún los servicios Webpay OneClick Mall.
+
+Una transacción OneClick Mall permite que el tarjetahabiente registre su
+tarjeta de la misma forma en que ocurre con una transacción OneClick, asociando
+dicha inscripción a un comercio padre. Ahora, una vez realizada la inscripción, 
+el comercio padre tiene permitido autorizar transacciones sin captura para
+los comercios “hijo” registrados que tengan habilitado captura diferida.
+Además posterior a la autorización tiene permitido capturar dicho monto reservado.
+La autorización se encarga de validar si es posible realizar el cargo a la
+cuenta asociada a la tarjeta de crédito realizando en el mismo acto la reserva
+de monto de la transacción.
+La captura hace efectiva la reserva hecha previamente o cargo en la cuenta de
+crédito asociada a la tarjeta del titular.
+Estas modalidades, por separado, solo son válidas para tarjetas de crédito.
+
+Para realizar esa captura explícita debe usarse el método `capture()`
+
+#### `capture()`
+
+Este método permite a los comercios OneClick Mall habilitados, poder
+realizar capturas diferidas de una transacción previamente autorizada. El método
+contempla una única captura por cada autorización. Para ello se deberá indicar los
+datos asociados a la transacción de venta y el monto requerido para capturar, el cual
+debe ser menor o igual al monto originalmente autorizado.
+Para capturar una transacción, ésta debe haber sido creada por un código de
+comercio configurado para captura diferida. De esta forma la transacción estará
+autorizada pero requerirá una captura explícita posterior para confirmar la
+transacción.
+
+
+**Parámetros**
+
+Nombre  <br> <i> tipo </i> | Descripción
+------   | -----------
+authorizationCode  <br> <i> xs:string </i> | Código de autorización de la transacción que se requiere capturar. Largo máximo: 6.
+buyOrder  <br> <i> xs:string </i> | Orden de compra de la transacción que se requiere capturar. Largo máximo: 26.
+commerceId  <br> <i> xs:long </i> | Código de tienda mall que realizó la transacción. Largo: 12.
+capturedAmount  <br> <i> xs:decimal </i> | Monto que se desea capturar. Largo máximo: 10.
+
+**Respuesta**
+
+Nombre  <br> <i> tipo </i> | Descripción
+------   | -----------
+token  <br> <i> xs:string </i> | Token de la transacción.
+authorizationCode  <br> <i> xs:string </i> | Código de autorización de la captura diferida.
+authorizationDate  <br> <i> xs:string </i> | Fecha y hora de la autorización.
+capturedAmount  <br> <i> xs:decimal </i> | Monto capturado.
+
+<aside class="notice">
+En caso de error apareceran los mismos códigos exclusivos del método `capture()`
+para captura simpultanea.
+</aside>
+
+### Anulación Monto Capturado Webpay OneClick Mall
+> Los SDKs no soportan aún los servicios Webpay OneClick Mall.
+
+Para anular una transacción se debe invocar al método `nullify()`.
+
+#### `nullify()`
```

Comment:
How is this different than the `nullify()` method documented above? (before the new documentation)

Reply:
See https://transbankdevelopers.cl/referencia/webpay#reversar-o-anular-un-pago-webpay-oneclick-mall

Reply:
It appears to be the same. The documentation I received just has a different explanation in the description field of the parameters, it mentions the capture functionality.
For example, "authorizedAmount" description:
```
Monto autorizado y capturado luego de invocar
el método capture(). Largo máximo: 10.
```

Everything else is pretty much the same, I will go ahead and use the `nullify` under 'Reversar o Anular un pago Webpay OneClick Mall'. Do you think I should merge the descriptions too?